index.js

```diff
@@ -178,7 +178,7 @@ export default (apiUrl, httpClient = fetchUtils.fetchJson, uploadFields = []) =>
         const { headers, json, total } = response;
         switch (type) {
 	    case GET_ONE:
-	        return { data: replaceRefObjectsWithIds(json) };
+	        return { data: json };
```

Comment:
Please revert this back to the original. _replaceRefObjectsWithIds_ was added to handle issues with Strapi Collections. Reference inputs would not be displayed correctly otherwise. Until we find a solution that works in harmony with Strapi Collections and Components, this should stay the way it is for now. Thanks!